Source/utils/lua.cpp

```diff
@@ -0,0 +1,57 @@
+#include "utils/lua.hpp"
+
+extern "C" {
+#include <lauxlib.h>
+#include <lua.h>
+#include <lualib.h>
+}

```

Comment:
It may be worth noting which version of Lua this is somewhere in this file and in CMakeLists.

Reply:
This would be an odd place to write that since this is not where the version would be updated and so would easily be missed.

The best way to see what is being linked is opening the hash from CMakeLists.txt
```
https://github.com/walterschell/Lua/archive/<hash>.tar.gz
```
```
https://github.com/walterschell/Lua/commit/<hash>
```

I have instead defined the somewhat standard `_VERSION` so you can print it in your script to find what version is running.

```lua
function GameStartedEvent()
    print("Hello from " .. _VERSION)
end
```

> Hello from Lua 5.4